store/player.ts

```diff
@@ -23,20 +23,28 @@ export const usePlayerStore = defineStore(
       }
     }
 
-    function resetGame(mode: 'tryAgain' | 'exit' = 'exit') {
+    function resetGame(mode: 'tryAgain' | 'exit' | 'resetGame' = 'resetGame') {
       const { resetStages } = useStageStore()
+      if (mode === 'resetGame') {
+        playerHealth.value = 0
+        router.push('/')
+      }
       if (mode === 'tryAgain') {
+        console.log('tryAgain')
         playerHealth.value = 100
         isGameInProgress.value = true
         router.push('/game/stage-hub')
+        resetStages()
+
       }
       if (mode === 'exit') {
         playerName.value = ''
         playerHealth.value = 0
         router.push('/')
+        localStorage.clear()
```

Comment:
gave the 'exit' aka 'quit the entire game' mode the responsibility to clear the localStorage because this is the behavior that is desired. we want to clear everything when the player wants to quit not when one if statement is executed